model/span.go

```diff
@@ -93,6 +94,30 @@ func (s *Span) IsRPCServer() bool {
 	return s.HasSpanKind(ext.SpanKindRPCServerEnum)
 }
 
+// GetStartTime returns the span's StartTime as time.Time value.
+func (s *Span) GetStartTime() time.Time {
+	seconds := s.StartTime / 1000000
+	nanos := 1000 * (s.StartTime % 1000000)
+	return time.Unix(int64(seconds), int64(nanos))
+}
+
+// GetDuration returns the span's duration as time.Duration value.
+func (s *Span) GetDuration() time.Duration {
+	return time.Duration(s.Duration * 1000)
```

Comment:
if `s.Duration` is in microseconds, then this should be `return time.Duration(s.Duration) * time.Microsecond`